src/main/groovy/com/readytalk/gradle/plugins/ci/CiPublishingPlugin.groovy

```diff
@@ -20,8 +20,9 @@ class CiPublishingPlugin implements Plugin<Project>, PluginUtils {
 
   void apply(final Project project) {
     this.project = project
+    project.plugins.apply(BasePlugin)
 
-    withAnyPlugin(['ivy-publish', 'maven-publish']) {
+    withAnyPlugin(['ivy-publish', 'maven-publish', 'com.gradle.plugin-publish']) {
```

Comment:
Yeah after looking at this more, I am starting to see how it makes things better. I agree I don't think it matters in this case, and I am glad to be more aware of how that helper function works. Thanks for calling it out.